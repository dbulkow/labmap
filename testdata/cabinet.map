lin302  lnx2 pos0 com1-no  com2-57600,8,1,N:/dev/ttyUSBX pdu5 kvm2
lin303  lnx3 pos0 com1-57600,8,1,N:/dev/ttyUSBY com2-no  pdu4 kvm3
