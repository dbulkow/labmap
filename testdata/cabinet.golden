{
    "status": "Success",
    "cabinet": {
        "vtm0": "lin302-vtm0",
        "vtm1": "lin302-vtm1",
        "cabinet": "2",
        "position": "0",
        "com1": "",
        "serial1": "",
        "params1": "",
        "com2": "telnet lnx2-debug 10002",
        "serial2": "/dev/ttyUSBX",
        "params2": "57600,8,1,N",
        "outlet": "5",
        "kvm": "lnx2-kvm",
        "pdu0": "lnx2-pdu0",
        "pdu1": "lnx2-pdu1"
    }
}